caddyconfig/httpcaddyfile/directives.go

```diff
@@ -37,10 +37,9 @@ import (
 // The header directive goes second so that headers
 // can be manipulated before doing redirects.
 var directiveOrder = []string{
+	"map",
 	"root",
-
 	"header",
-
 	"redir",
```

Comment:
Restore this spacing - not sure why it has to be removed?

Reply:
Reinstated, I wasn't sure why they were there :)